packages/components/src/custom-select-control/index.js

```diff
@@ -77,6 +79,12 @@ export default function CustomSelectControl( {
 		stateReducer,
 	} );
 
+	const describedByBuild = describedBy
+		? describedBy
+		: sprintf(
```

Comment:
This makes linter unhappy 😄

Reply:
The auto-format keeps changing it back. How should it look?

Reply:
I think I got it now.